contracts/integrations/test/actors/base.ts

```diff
@@ -62,6 +64,34 @@ export class Actor {
         );
     }
 
+    /**
+     * Mints some number of ERC721 NFTs and approves a spender (defaults to the ERC721 asset proxy)
+     * to transfer the token.
+     */
+    public async configureERC721TokenAsync(
+        token: DummyERC721TokenContract,
+        spender?: string,
+        numToMint?: number,
+    ): Promise<BigNumber[]> {
+        const tokenIds: BigNumber[] = [];
+        _.times(numToMint || 1, async () => {
```

Comment:
What if we want to mint no tokens?

Reply:
all I want for christmas is nullish coalesce in our TS